tox.ini

```
@@ -36,10 +36,8 @@ deps =
     testing.postgresql
     testing.mysqld
     webtest
-    mock
-
     # Python2 only deps
-    py{27}: enum34
+    py{27}: enum34, mock
```

Sorry my bad I think there should not be a comma for pip dependencies, committing this change:

```suggestion
    py{27}: enum34 mock
```